OrbitQt/CallTreeWidget.cpp

```diff
@@ -103,6 +99,40 @@ void CallTreeWidget::SetBottomUpView(std::unique_ptr<CallTreeView> bottom_up_vie
   ui_->callTreeTreeView->hideColumn(CallTreeViewItemModel::kExclusive);
 }
 
+void CallTreeWidget::resizeEvent(QResizeEvent* /*event*/) {
+  if (column_resizing_state_ == ColumnResizingState::kInitial) {
+    column_resizing_state_ = ColumnResizingState::kWidgetSizeSet;
+    ResizeColumnsIfNecessary();
+  }
+}
+
+void CallTreeWidget::ResizeColumnsIfNecessary() {
+  // Resize columns only once, when the following holds:
+  // - a non-empty CallTreeView is set;
+  // - the size of the QTreeView has been set, which means that the size of CallTreeWidget has been
+  //   set, which in turns means that the tab has been shown at least once.
+  if (column_resizing_state_ == ColumnResizingState::kWidgetSizeSet &&
+      hooked_proxy_model_ != nullptr && hooked_proxy_model_->rowCount(QModelIndex{}) > 0) {
```

Comment:
nit: inverse the if - use early return here.

Reply:
Good point, done.